sdk/lib/opentelemetry/sdk/internal.rb

```diff
@@ -20,8 +20,12 @@ def valid_key?(key)
         key.instance_of?(String)
       end
 
+      def valid_numeric?(value)
```

Comment:
Maybe try to be consistent with `boolean?` above?
```suggestion
      def numeric?(value)
```